code/game/machinery/station_map.dm

```diff
@@ -329,3 +329,93 @@ var/list/station_holomaps = list()
 	legend.pixel_x = 7*WORLD_ICON_SIZE
 	legend.pixel_y = 7*WORLD_ICON_SIZE
 	station_map.overlays |= legend
+
+/obj/machinery/station_map/strategic
+	name = "strategic station holomap"
+	density = 1
+	machine_flags = null
+	icon = 'icons/obj/stationobjs_64x64.dmi'
+	pixel_x = -1*WORLD_ICON_SIZE/2
+	pixel_y = -1*WORLD_ICON_SIZE/2
+
+	var/list/watching_mobs = list()
+	var/list/watcher_maps = list()
+
+/obj/machinery/station_map/strategic/New()
```

Comment:
The rule has ALWAYS been if it holds references it needs to have those references removed in destroy